debian/libplank-doc.install

```diff
@@ -1,2 +1 @@
-#docs/c-doc/html/* usr/share/gtk-doc/html/plank-c
-usr/share/gtk-doc/html/plank
+usr/share/doc/plank/* usr/share/gtk-doc/html/plank
```

Comment:
The changes to this file need to be reverted as the meson port installs the docs directly to the correct path, though I guess we could drop the commented line.

Reply:
Confirmed that a local build works on `focal` after reverting this change, will test bionic next.